phaseshifts/backends.py

```
@@ -29,59 +29,155 @@ def autogen_from_input(self, bulk_file, slab_file, tmp_dir=None, **kwargs):
+        firstline = result[0]
+        phaseshifts = result[1]
+        viper_phaseshifts.writePHASESHIFTS(
+            firstline, phaseshifts, file_path=output_path
+        )
+    finally:
+        os.chdir(cwd)
+
+    return [output_path]
+
+
+def _run_native_eeasisss(parameters_file, workdir, output_file):
```

Ruff (0.14.10)

Missing return type annotation for private function `_run_native_eeasisss`

(ANN202)